cupy/random/generator.py

```diff
@@ -201,6 +201,20 @@ def lognormal(self, mean=0.0, sigma=1.0, size=None, dtype=float):
             func = curand.generateLogNormalDouble
         return self._generate_normal(func, size, dtype, mean, sigma)
 
+    def power(self, a, size=None, dtype=float):
+        """Returns an array of samples drawn from the power distribution.
+
+        .. seealso::
+            :func:`cupy.random.power` for full documentation,
+            :meth:`numpy.random.RandomState.power`
+        """
+        a = cupy.asarray(a)
+        x = self.standard_exponential(size=size, dtype=dtype)
+        cupy.exp(-x, out=x)
+        cupy.add(1, -x, out=x)
+        cupy.power(x, 1./a, out=x)
```

Comment:
Return `0` for `a = 0`.

Reply:
Already, this function return `0` for `a=0` because `cupy.power(x, cupy.inf)==0` for `abs(x)<1`.